app/helpers/application_helper.rb

```diff
@@ -40,7 +40,7 @@ def default_title
                     scope: [:controller, :action, name],
                     default: action.humanize)
 
-    "#{controller_name} - #{action_name}"
+    "#{controller_name == "Services" ? "Products" : controller_name} - #{action_name}"
```

Comment:
Can you create a test for that?
I think this will not work because of the capitalized words